chef/cookbooks/ceph/providers/client.rb

```diff
@@ -67,6 +67,12 @@ def auth_set_key(ceph_conf, admin_keyring, keyname, caps)
   unless get_or_create.stderr.scan(/EINVAL.*but cap.*does not match/).empty?
     Chef::Log.info("Deleting old key with incorrect caps")
     # don't remove custom existing caps for osd pools
+    cur_caps = get_caps(ceph_conf, admin_keyring, keyname)
+    if cur_caps.empty?
+      caps_osd = caps["osd"]
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `caps_osd`. Did you mean `caps`? (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)